### ISSUE TITLE
CMake script compatibility with Mac OSX

### ISSUE THREAD

**Issue by jayten** [FEST3D/FEST-3D#3]
The CMake script behaves erratically when used on Mac OSX. It is either not able to find the correct configuration or does not exit the process (**execute_process** in CMake). The test was performed on OS version 10.13.4 (High Sierra) with following installed using Homebrew
- CMake-3.15.0
- Gcc-9.1.0
- Libmpc-1.1.0
- Mpfr-4.0.2
- Isl-0.21
- Gmp-6.1.2_2
- mpich-3.3.1

**Comment by jayten** [comment]
The following piece of code in the **SetFortranFlags.cmake** module is creating problem:
```
# Don't add underscores in symbols for C-compatability
SET_COMPILE_FLAG(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS}"
                 Fortran "-fno-underscoring")
```

**Comment by jayten** [comment]
The issue has been sorted and corrected in the #4.
